### ISSUE TITLE
Suggestion for Data Dictionary Register Proposals

### ISSUE THREAD

**Issue by JeffWootton** [iho-ohi/S100Infrastructure#27]
I have recently rejected a proposal from the S-124 Project Team to supersede the modelling of the complex attribute `featureReference` in the Data Dictionary Register, including the removal of some sub-attributes. This complex attribute had been originally proposed by the WMO as a complex attribute for their S-412 Weather Product Specification. I rejected this proposal as this may have impacted on the WMO Product Specification and there had been no consultation between the WWNWS (as developers of S-124) and the WMO.

I think it would therefore be useful to have a pop-up message appear on the screen when a Proposer is proposing a change (Clarification or Supersession) to an item in the Data Dictionary Register that was originally proposed (and accepted) by another Domain. The message could be similar to: 
"_This item has been registered under the [Domain name] Domain. Please consult with the [Domain name] Domain representatives prior to submission of this proposal to agree that the change to be made is acceptable for all parties._".

If possible, this change should also be considered for change proposals to the Concept Register where a change is being proposed for a concept that has been modelled by a different Domain in the Data Dictionary Register.

**Comment by dyfaos3** [comment]
We would be grateful if you could kindly provide clear guidance on this matter. 

Based on your guidance, we will review our internal circumstances and determine the appropriate way forward.

**Comment by JeffWootton** [comment]
In addition to the information included in the first comment against this Issue:
- Proposers should be provided with a message advising them to consult with representative(s) of the Domain that originally proposed the item in the Data Dictionary Register before proceeding with their Clarification or Supersession proposal as their proposal may impact on the Product Specification(s) that have been developed by that Domain to reflect their data model. The message should additionally include a statement that the Proposer should include a comment with their proposal (in the Justification) that this consultation has taken place.
- A similar message should be provided to a Proposer that is proposing a change (Clarification or Supersession) to a proposal in the Concept Register that was originally submitted by another Submitting Organisation.
- Taking Raphael's comment into account, I agree that it would also be useful for the original submitter of the proposal to be sent an email when another Submitting Organisation is proposing a change to one of their proposals.

**Comment by dyfaos3** [comment]
@JeffWootton 
Then, would it be acceptable to implement a feature where a pop-up appears upon entering the Clarification or Supersession page, asking whether prior consultation has taken place? 

If the user clicks 'Yes', they can proceed with the task, and if they click 'No', they will be redirected to the previous page. 

Furthermore, once the proposal is submitted, an automated email would be sent to the original proposer detailing who made the modifications and which specific parts were changed. 

Would this approach be satisfactory?

**Comment by JeffWootton** [comment]
Yes - if the proposer of the Clarification or Supersession is not a member of the same Domain as the original proposer.

**Comment by dyfaos3** [comment]
@JeffWootton 
The updates have been deployed to the production environment. 

Please feel free to contact me if you experience any issues.
